optaplanner-core/src/main/java/org/optaplanner/core/impl/exhaustivesearch/DefaultExhaustiveSearchPhase.java

```diff
@@ -269,4 +243,35 @@ public void solvingEnded(SolverScope<Solution_> solverScope) {
         decider.solvingEnded(solverScope);
     }
 
+    public static class Builder<Solution_> extends AbstractPhase.Builder<Solution_> {
```

Comment:
Do we need a builder pattern?

Reply:
It depends on if all the properties are mandatory. If they are not, removing setters leads to a combinatorial explosion of constructors.